core/src/main/resources/jenkins/install/pluginSetupWizard_zh_CN.properties

```diff
@@ -0,0 +1,98 @@
+# The MIT License
+#
+# Copyright (c) 2017-, vivo.com
+# Chinese (Simplified) translated by Gentle Yang
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+installWizard_welcomePanel_title=\u65b0\u624b\u5165\u95e8
+installWizard_welcomePanel_banner=\u81ea\u5b9a\u4e49jenkins
+installWizard_welcomePanel_message=\u63d2\u4ef6\u901a\u8fc7\u9644\u52a0\u7279\u6027\u6765\u6269\u5c55jenkins\u4ee5\u6ee1\u8db3\u4e0d\u540c\u7684\u9700\u6c42\u3002
+installWizard_welcomePanel_recommendedActionTitle=\u5b89\u88c5\u63a8\u8350\u7684\u63d2\u4ef6
+installWizard_welcomePanel_recommendedActionDetails=\u5b89\u88c5jenkins\u793e\u533a\u63a8\u8350\u7684\u63d2\u4ef6\u3002
+installWizard_welcomePanel_customizeActionTitle=\u9009\u62e9\u63d2\u4ef6\u6765\u5b89\u88c5
+installWizard_welcomePanel_customizeActionDetails=\u9009\u62e9\u5e76\u5b89\u88c5\u6700\u9002\u5408\u7684\u63d2\u4ef6\u3002
+installWizard_jenkinsVersionTitle=Jenkins 
+installWizard_offline_title=\u79bb\u7ebf
+installWizard_offline_message=\u8be5jenkins\u5b9e\u4f8b\u4f3c\u4e4e\u5df2\u79bb\u7ebf\u3002\
+<p style="font-size:18px; margin-top: 6%"> \
```

Comment:
Wow, it is really being stored in localization files 🤦‍♂️

Reply:
Should I remove all HTML in the property file when translate ?  Include <a> tag ?

Reply:
It's not your problem, I'd say. It was introduced by the Setup Wizard developers. I would keep it as is in the current pull request.